R/skip.R

```diff
@@ -163,6 +166,17 @@ skip_on_appveyor <- function() {
   skip("On Appveyor")
 }
 
+#' @export
+#' @rdname skip
+skip_on_ci <- function() {
+  ci <- Sys.getenv("CI")
+  if (!any(identical(ci, "true"), identical(ci, "True"))) {
```

Comment:
```suggestion
  if (!isTRUE(as.logical(Sys.getenv("CI")))) {
```

Reply:
This is a potential shorter / more robust solution.

``` r
isTRUE(as.logical("true"))
#> [1] TRUE
isTRUE(as.logical("True"))
#> [1] TRUE
isTRUE(as.logical("False"))
#> [1] FALSE
isTRUE(as.logical(""))
#> [1] FALSE
isTRUE(as.logical(NULL))
#> [1] FALSE
isTRUE(as.logical(NA))
#> [1] FALSE
```

<sup>Created on 2018-12-04 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>